client/scala/src/main/scala/ai/verta/swagger/client/HttpClient.scala

```diff
@@ -31,20 +31,21 @@ class HttpClient(val host: String, val headers: Map[String, String]) {
         .reduce((p1, p2) => p1 + "&" + p2)
   }
 
-  def request[T1, T2](method: String, path: String, query: Map[String, List[String]], body: T1, parser: JValue => T2)(implicit ec: ExecutionContext, m: Manifest[T2]): Future[Try[T2]] = {
+  def request[T1, T2](method: String, path: String, query: Map[String, List[String]], body: T1, parser: JValue => T2, auth: Option[(String, String)] = None)(implicit ec: ExecutionContext, m: Manifest[T2]): Future[Try[T2]] = {
```

Comment:
I'd highly recommend using a case class instead of a tuple. Always aim for easier consumption.

Reply:
Fixed. I just implemented BasicAuthorization case class.